tests/storage_test.go

```diff
@@ -89,9 +100,20 @@ var _ = Describe("Storage", func() {
 		// Let's get the IP of the pod of the VM
 		pods, err := coreClient.CoreV1().Pods(tests.NamespaceTestDefault).List(services.UnfinishedVMPodSelector(vm))
 		//FIXME Sometimes pods hang in terminating state, select the pod which does not have a deletion timestamp
-		podIP := ""
+		hostIP := ""
 		for _, pod := range pods.Items {
 			if pod.ObjectMeta.DeletionTimestamp == nil {
+				hostIP = pod.Status.HostIP
```

Comment:
The check for the actual IP, now gets very difficult. Depending on the kube proxy mode, in which network namespace the qemu process is, which network provider is used, or  if just bridging is used, we always see different IPs connecting on the iscsi target. The BeforeEach block already makes sure that there is no connection to the iscsi target before each test. Maybe we should just check if a connection appears, and not try to check for specific IPs.